frontend/src/hooks/api/tags/types.ts

```diff
@@ -18,6 +18,7 @@ export type CreateTagDTO = {
   tagSlug: string;
   tagName: string;
   tagColor: string;
+  checkedSecrets: { _id: string, isChecked: string | boolean }[]
```

Comment:
I dont see the checkedSecret field in Tag model. Is this in create

Reply:
I would need to add it there also

Reply:
Addressed!